### PR TITLE
document where the chart is hosted

### LINKED ISSUE #552: Host your operator Chart in some repo
We're currently just manually pulling in your chart into another chart of ours to use it as a subchart. We'd love to be able to just include the chart here as a dependency in  requirements.yaml by pulling from a published repo location as opposed to manually pulling in new versions when they're available. Let us know if you think this would be possible. Thanks.

### PR DESCRIPTION
**Pull request by yelhouti**
fixes #552

### CODE REVIEW

**Comment by FxKu** [comment]
:+1:

**Comment by RafiaSabih** [comment]
:+1:

**Comment by yelhouti** [comment]
can anyone retest please, CI is trolling

File: docs/quickstart.md [hunk 1]
```diff
@@ -85,6 +85,8 @@ The chart works with both Helm 2 and Helm 3. The `crd-install` hook from v2 will
 be skipped with warning when using v3. Documentation for installing applications
 with Helm 2 can be found in the [v2 docs](https://v2.helm.sh/docs/).
 
+The chart can also be found at: https://opensource.zalando.com/postgres-operator/charts/postgres-operator/
```
Comment (ePaul) [inline docs/quickstart.md:88]: This URL currently gives a 404. Is this something which is supposed to work after this is merged?

Comment (FxKu) [reply docs/quickstart.md:88]:
```suggestion
The chart can also be found at: https://opensource.zalando.com/postgres-operator/charts/postgres-operator/postgres-operator-1.6.3.tgz
```

Comment (yelhouti) [reply docs/quickstart.md]: this is the url of the chart repo, helm uses it to query the index.yaml: 
```
https://opensource.zalando.com/postgres-operator/charts/postgres-operator/index.yaml
```
and from there, retrieve the path of the chart and append it to the same prefix to download.

**Comment by yelhouti** [comment]
@FxKu please revert your change

File: docs/quickstart.md (hunk 1 above)
Comment (yelhouti) [reply docs/quickstart.md]: @FxKu please revert your change

Comment (yelhouti) [reply docs/quickstart.md]: the way you install chart is for example:
```
helm repo add postgres-operator https://opensource.zalando.com/postgres-operator/charts/postgres-operator/
helm install postgres-operator/postgres-operator postgres-operator ...
```
If you do not want the 404 you can make it redirect to the github repo are something like that, but it is really not a problem

Comment (FxKu) [reply docs/quickstart.md]: Then I suggest these are the commands you shall add to the quickstart. Otherwise it's confusing.

Comment (yelhouti) [reply docs/quickstart.md]: @FxKu this standard when using helm it is also not the only way, if you use helmfile or other your file looks like:
repositories:
  - name: postgres-operator
    url: https://opensource.zalando.com/postgres-operator/charts/postgres-operator/
releases:
  - chart: postgres-operator/postgres-operator
    namespace: postgres-operator
    version: 1.6.3
    name: postgres-operator

File: docs/quickstart.md [hunk 2]
```diff
@@ -85,6 +85,8 @@ The chart works with both Helm 2 and Helm 3. The `crd-install` hook from v2 will
 be skipped with warning when using v3. Documentation for installing applications
 with Helm 2 can be found in the [v2 docs](https://v2.helm.sh/docs/).
 
+The chart can also be found at: https://opensource.zalando.com/postgres-operator/charts/postgres-operator/postgres-operator-1.6.3.tgz
```
Comment (FxKu) [inline docs/quickstart.md:88]:
```suggestion
The chart is also hosted at: https://opensource.zalando.com/postgres-operator/charts/postgres-operator/
```

File: docs/quickstart.md (hunk 1 above)
Comment (yelhouti) [reply docs/quickstart.md]: @FxKu that is good with me :) thanks
Somehow CI keeps failing though...

**Comment by FxKu** [comment]
:+1:

**Comment by RafiaSabih** [comment]
:+1:

**Pull request merged**